### ISSUE TITLE
[Feature Request]: Emergency Garbage Clean

### ISSUE THREAD

**Issue by Lazejun** [official-antistasi-community/A3-Antistasi#1127]
Basicly the same as the Normal Garbage Clean
just bound to the HQ Marker and Limited in Range.

This might be something for Later -------> UI Rework since it would at a Button in our Limited UI.

A Possible Radius/Range would be 100 Meters since it enough range to cover a lot,
but there is also the possibility of Ace-Cookoff still hitting things from further than 100 Meters.

Personally i think 100 Meter is sufficient.

**Comment by Bob-Murphy** [comment]
> This might be something for Later -------> UI Rework since it would at a Button in our Limited UI.

Exactly. If you would have not added this comment, I would have done it.

**Comment by jaj22** [comment]
My personal inclination:

1. Add another garbage clean specifically for dead vehicles. That's harmless even if applied over the whole map.
2. Put a delay on both garbage cleans to avoid cheating.

**Comment by MeltedPixel** [comment]
Bob would like Frost to take a look at this one next time he has some of the valuable resource... Time...

**Comment by Tiny-DM** [comment]
Added in #3515
